_data/guides.yaml

```diff
@@ -28,6 +28,9 @@ categories:
       - title: Identity Propagation with OpenID Connect
         url: /guides/security-oidc-identity-propagation
         description: Learn how to propagate identities within a deployment and across deployments when securing apps with OpenID Connect.
+      - title: Securing a WildFly Application Using OpenID Connect With Additional Scope Values
+        url: /guides/elytron-oidc-client-scope
```

Comment:
To be consistent with the other security guide URLs, let's use `/guides/security-oidc-scope`

Reply:
s/Securing a WildFly Application/Securing WildFly Apps

Reply:
fixed!